### ISSUE TITLE
There should be a  doc-2.2 branch

### ISSUE THREAD

**Issue by DrDaveD** [cvmfs/doc-cvmfs#33]

**Comment by reneme** [comment]
We are moving away from LaTeX and will continue to write the documentation in [Sphinx](http://www.sphinx-doc.org/en/stable/) using [reStructured Text](http://docutils.sourceforge.net/rst.html). This gives us the flexibility to provide all kinds of document formats -- most notably HTML (as seen in [this demo](http://ecsft.cern.ch/dist/cvmfs/doctest/index.html)).

What comes closest to a `doc-2.2` branch is the `sphinx` branch that was created last week.

**Comment by DrDaveD** [comment]
The 'sphinx' name for the branch without a number is a bit short-sighted since in the future there will presumably be other releases.

**Comment by reneme** [comment]
It's work in progress and subject to change. Anyway, the plan is to mark documentation versions with tags rather than branches.

**Comment by reneme** [comment]
I've now updated this repository to it's final form. LaTeX sources are gone (but of course reachable by tags and `legacy-***` branches) and replaced by reStructured Text.

We will use `master` for cutting edge editing and branch off specific (released) versions of CernVM-FS. For the moment there is a `2.1.20` branch that forms the base-line for the HTML-based documentation. The `2.2.0` branch will follow as soon as we have edited all necessary pieces.

There is a testing version on [readthedocs](http://cvmfs-test.readthedocs.org) that show cases the versioned documentation (lower left).
